src/components/World/World.js

```diff
@@ -3,6 +3,9 @@ import ReactDOM from 'react-dom';
 
 import { createPureComponent } from 'utils/createPureComponent';
 import { gridCoordsToOffsetStyle } from 'utils/gridCoordsToStyle';
+import hasEmoji from 'utils/hasEmoji';
+
+import classNames from 'classnames';
```

Comment:
Nitpick, but can we move this below `ReactDOM` with newlines around it? The rest of the codebase `import`s third-party stuff at the top, and they're grouped by dependency. See [this example](https://github.com/stevenhauser/i-have-to-return-some-videotapes/blob/master/src/components/EditorTool/EditorTool.js#L1-L6) for reference.